lib/client/api/sharing.js

```diff
@@ -142,6 +142,20 @@ module.exports = function (jiffClient) {
     return jiffClient.open(imitationSecretShare, receivers, op_id);
   };
 
+  /**
+   * Receive arrays of shares from the specified parties and reconstruct their secrets.
+   * Use this function in a party that will receive some answer/value but does not have a share of it.
+   * @method receive_open
+   * @memberof module:jiff-client~JIFFClient
+   * @instance
+   * @param {Array} senders - an array with party ids (1 to n) specifying the parties sending the shares, this must be provided!
+   * @param {number} [threshold=senders.length] - the min number of parties needed to reconstruct the secret, defaults to all the senders
+   * @param {number} [Zp=jiff_instance.Zp] - the mod (if null then the default Zp for the instance is used)
+   * @param {string|number} [op_id=auto_Gen()] - same as jiff-instance.open
+   * @returns {!promise} a (JQuery) promise to the open value of the secret.
+   */
+  jiffClient.receive_open_array = arraysSharing.jiff_receive_open_ND_array.bind(null, jiffClient);
```

Comment:
the docs seem off a bit here. 
@method should say receive_open_array

also, I thought we should have a receivers array similar to senders, otherwise op_ids will not be set up correctly. See https://multiparty.org/jiff/docs/jsdoc/module-jiff-client-JIFFClient.html#receive_open

Reply:
I think this same comment applies to jiff_receive_open_ND_array